dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -383,6 +384,7 @@ def profile(self) -> dict:
             skewness=self.np_type_to_type(self.skewness),
             kurtosis=self.np_type_to_type(self.kurtosis),
             histogram=self._get_best_histogram_for_profile(),
+            num_quantiles=self.num_quantiles,
```

Comment:
let's not return in the profile itself as this is a setting vs a profiled value.

Reply:
+1

Reply:
@clee1152 just bumping in case this was missed.